src/helpers/data.ts

```diff
@@ -63,7 +63,9 @@ export function load(
       buttons: [],
       notes: stringToTrimmedArray(row.notes, true),
       search: '',
-      tags: []
+      tags: [],
+      meetingId : row.meeting_id,
+      meetingEmail : row.email
```

Comment:
would be more consistent for the keys to just be `id` and `email`

Reply:
done - changed to meetingName to "name" , meetingId to "id",  and meetingEmail to "email"